charts/rstudio-pm/README.md

```diff
@@ -119,7 +141,7 @@ The Helm `config` values are converted into the `rstudio-pm.gcfg` service config
 | pod.labels | object | `{}` | Additional labels to add to the rstudio-pm pods |
 | pod.lifecycle | object | `{}` | Container [lifecycle hooks](https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/) |
 | pod.securityContext | object | `{}` | the [securityContext](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/) for the pod |
-| pod.serviceAccountName | bool | `false` | serviceAccountName is a string representing the service account of the pod spec |
+| pod.serviceAccountName | string | `""` | Deprecated, use `serviceAccount.name` instead |
```

Comment:
Should we warn about this if it is used, to make the deprecation a bit more formal? If we throw it in `NOTES.txt`, that could be a good place to establish this as a convention for how to migrate.

Reply:
Good idea.